app/models/like.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: likes
+#
+#  id         :bigint           not null, primary key
+#  user_id    :bigint           not null
+#  post_id    :bigint           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: :user_id
+  belongs_to :post, foreign_key: :post_id
+
+  after_create :update_post_likes_counter
+  after_destroy :update_post_likes_counter
+
+  def update_post_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the `#update_posts_likes_counter` method every time a new `like` has been saved to the database, then make the `update_posts_likes_counter` method private

Reply:
Implemented the change. Thanks again @DeliceLydia